bootstrap/pkg/utils/iamutils.go

```diff
@@ -82,6 +69,31 @@ func GetIamPolicy(project string) (*cloudresourcemanager.Policy, error) {
 	return service.Projects.GetIamPolicy(project, req).Context(ctx).Do()
 }
 
+// Gets IAM plicy from GCP for the whole project.
```

Comment:
what does cleared mean? Could you explain in comment?

Reply:
updated